grammar.js

```diff
@@ -806,6 +1056,21 @@ function syn_sub(sub, ...args) {
   }
 }
 
+function syn_arg(arg, ...args) {
+  if (args.length > 0)
+    return seq(field('name', arg), token.immediate('='), field('val', ...args));
+  else
+    return field('name', arg);
+}
+
+function syn_sync_meth(arg, ...args) {
```

Comment:
I think we can survive two more characters there :wink: 
```suggestion
function syn_sync_method(arg, ...args) {
```